lib/matplotlib/streamplot.py

```diff
@@ -20,7 +20,7 @@
 
 
 def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
-               cmap=None, norm=None, arrowsize=1, arrowstyle='-|>',
```

Comment:
Changing the defaults is an API change which we should only do with a very good reason.